backend/database/migrations/2022_05_12_054522_add_created_by_and_updated_by_to_submissions.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+use App\Traits\WithConnectionName;
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    use WithConnectionName;
+
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('submissions', function (Blueprint $table) {
+            $createdBy = $table->unsignedBigInteger('created_by');
+            $updatedBy = $table->unsignedBigInteger('updated_by');
+
+            //SQLite doesn't like non-nullable columns having a NULL default when altering tables
```

Comment:
Is the creation of a submission considered an update and that's why `updated_by` is considered non-nullable?

Reply:
Correct. Just like updated_at.